packages/core/core/src/useTablePreferences/index.ts

```diff
@@ -62,13 +62,13 @@ export default function useTablePreferences() {
    * @param {string} tableKey The table identifier
    * @returns {TablePreferences} The stored preferences for the given tableKey
    */
-  const getTablePreferences = (tableKey: string): TablePreferences => {
+  const getTablePreferences = (tableKey: string, incomePreferences?: TablePreferences): TablePreferences => {
```

Comment:
Shall we rename this param to `givenPreferences` or `fallbackPreferences`? `income` sounds like money.

Reply:
Change it to `fallbackPreferences`